moveit_ros/benchmarks/examples/demo2.yaml

```diff
@@ -27,3 +27,4 @@ benchmark_config:
         - plugin: my_plugin_ns/my_plugin
           planners:
             - MyPlanner
+
```

Comment:
please don't introduce changes like this into PRs since nothing else in this file was modified

Reply:
sure this was my mistake, removed this change